examples/3.0-beta/bbba8553-8ec1-445f-82c9-a57251dd731c.json

```diff
@@ -0,0 +1,31 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.blockcerts.org/schema/3.0-beta/context.json",
```

Comment:
we should use https://w3id.org/blockcerts/3.0-beta url.
Btw we need to publish the context here too: https://www.blockcerts.org/schema/3.0-beta/context.json